plugins/Goals/Reports/Get.php

```diff
@@ -262,7 +262,7 @@ public function configureView(ViewDataTable $view)
                 };
             }
         } elseif ($view->isViewDataTableId(Evolution::ID)) {
-            if (!empty($idSite) && Piwik::isUserHasWriteAccess($idSite)) {
+            if (!empty($idSite) && Piwik::isUserHasWriteAccess($idSite) && $idGoal > GoalManager::IDGOAL_ORDER) {
```

Comment:
This fixes a "bug", where the ecommerce goal title was clickable, referring to an invalid page, as the ecommerce goal isn't editable.